src/components/WalletButton.tsx

```diff
@@ -32,11 +32,19 @@ const WalletButtonWrapper = ({
   return (
     <Button
       variant="outlined"
-      sx={{padding: "10px 25px", backgroundColor: theme.palette.mode === "dark" ? "#1B1F1E" : "white",color: theme.palette.mode === "dark" ? "#2ED8A7" : "#121615"}}
```

Comment:
Can you remove this file from the PR? There's at least two other PRs right now that have this file open.

Reply:
will do